### ISSUE TITLE
Is it iFrame or maybe iframe?

### ISSUE THREAD

**Issue by oldrup** [equalizedigital/accessibility-checker#770]
While attempting a danish translation for Accessibility Checker, I stumbled over this error description (screenshot below), warning me that an iFrame element title was missing. I'm almost sure that the HTML element is called iframe, not iFrame - the latter looking like an apple branded thing.

MDN : https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe
W3C: https://www.w3.org/TR/2011/WD-html5-20110525/the-iframe-element.html
EDAC: https://equalizedigital.com/accessibility-checker/iframe-missing-title/

![image](https://github.com/user-attachments/assets/c9a86d3d-d667-453e-80bf-27193c7e0c96)

What do you think?
